src/components/general/ErrorMessage/index.tsx

```diff
@@ -109,6 +110,7 @@ class ErrorMessageBoundry extends Component<ErrorMessageProps> {
     state = { didCatch: false };
     render() {
         if (this.state.didCatch) {
+            const styles = useStyles();
```

Comment:
Hooks should not be used in class components, should also not be rendered conditionally